index.js

```diff
@@ -74,6 +79,10 @@ module.exports = (dest, opts) => {
 			newPath = gutil.replaceExtension(newPath, opts.extension);
 		}
 
+		if (opts.transform) {
+			newPath = opts.transform(newPath) || '';
```

Comment:
To prevent a cold error when the provided function does not return a string. Do you think an error is the preferred way?